test/screenshot/config.js

```diff
@@ -324,3 +328,216 @@ for (let color of getButtonConfig('paypal', 'colors')) {
         }
     });
 }
+
+
+/** INSTALLMENT BUTTON CONFIG **/
+
+for (let locale of INSTALLMENT_BTN_LOCALE) {
```

Comment:
So, one thing about these tests is they're really slow. So maybe let's not test every locale and combination. Can we, for now, maybe just render two buttons -- one with installment period and one without?

Also can we get one button for each of the new logos?

Reply:
ok